.github/workflows/release.yaml

```diff
@@ -20,6 +20,11 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Set up node
```

Comment:
Shouldn't we separate this from the python release? In principle, they don't need to run in the same job, right?

Reply:
Not sure about the best practices in this case. If we already do it like this in libqasm, lets just keep it as is.